bower.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ember-metrics",
   "dependencies": {
-    "ember": "1.13.7",
+    "ember": "components/ember#release",
```

Comment:
@timini  In the future, major releases of Ember will likely be incompatible with this library. We're using [Semantic Versioning](http://semver.org/), so we want to watch a specific version of Ember.